lib/chitter.rb

```diff
@@ -0,0 +1,44 @@
+require 'pg'
+
+class Chitter
+
+  attr_reader :peep, :username, :name, :emailaddress, :password
+  
+  def initialize(peep:, username:, name:, emailaddress:, password:)
```

Comment:
I'm not sure why Chitter can be initialised with these values - is this initialize method used at all?